std_msgs/Header header

string[] name
float64[] q
float64[] qdot
float64[] dq
float64[] dqdot
float64[] effort_command
float64[] Kp
float64[] Kd
